experimental/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -59,7 +60,9 @@ export class TinyliciousClient {
             runtimeFactory,
             true,
         );
-        return this.getFluidContainerAndServices(container);
+        const resources = await this.getFluidResources(container);
+        const attach = async () => container.attach({url: serviceContainerConfig.id});
```

Comment:
Let's not split the attach off of the container.  It's the container getting attached, so it's more semantically appropriate to `container.attach()` than `resources.attach()`.

FWIW we should probably also reconsider the "resources" naming (not necessarily this PR, but in general).  I think the intent was to try to find a name that would make some sense describing both the `FluidContainer` as well as the services, but I think it encourages the object to "stick around" or have an identity of its own (like gaining an `attach()` method.  Whereas I think we don't want that, we want the caller of `getFluid***()` to immediately deconstruct the return value and work on the `FluidContainer` and services objects directly.  So I'd probably suggest removing the `TinyliciousResources` entirely and instead just return an anonymous object type.

Reply:
@ChumpChief , I wasn't really apart of the tuple -> object conversation but if the goal is to have an anonymous object type that the user immediately deconstructs I think the tuple model works better. I'm also not sure I like the attach on the FluidContainer object itself since it's a create specific concept.

Reply:
but I agree `resources.attach()` is odd.

Reply:
revamped all of this